### ISSUE TITLE
Some SDFat Examples do not work

### ISSUE THREAD

**Issue by FrankBoesing** [PaulStoffregen/SD#32]
See title :)

**Comment by PaulStoffregen** [comment]
Yup, saw this with my script which recompiles all examples from all libraries.  Lots of failed compiles because SdFat examples depend on the File typedef which isn't used when FS.h is present.

It's not going to be fixed for the upcoming 1.55 release.

For 1.56, I'm consider a few options...

1: Rename "File" to "SdFile" in all those examples
2: Just delete those examples
3: Fork SdFat and edit the code to return core library File class
4: Try to collaborate with Bill to better integrate with Teensy core library
